test/test.js

```diff
@@ -14,10 +18,10 @@ describe('return type check', () => {
     });
 
     it('Check 200 http and Name', async () => {
-        const answer = await got('http://localhost:8000/');
+        const answer = await got(`http://${defaultValues.adress}:${defaultValues.port}`);
```

Comment:
That is a very good example of how we should use default values in tests, good job! :+1: